### DISCUSSION TITLE
OneDrive sync issues

### DISCUSSION THREAD

**Discussion by vicar5235** [spencerwooo/onedrive-vercel-index#416] [Q&A] [answered]
Hello, I have encountered a problem. After the deployment is completed, I updated the file in OneDrive, but the network disk has not been synchronized. How to solve this?

Must need to redeploy, he will show the latest file...

![image](https://user-images.githubusercontent.com/35890470/153693076-a51a95eb-78cf-467e-9957-8a5173b6a818.png)
![image](https://user-images.githubusercontent.com/35890470/153693087-5942872f-d625-49e4-8a57-5ef22eaf471a.png)

**Comment by spencerwooo** [comment, marked as answer]
https://github.com/spencerwooo/onedrive-vercel-index/discussions/407

> The cache is valid for 10 minutes on the edge. - This means you won't see your newly added files for at least 10 minutes after your files are done uploading. 10 minutes is applied as we consider your data in OneDrive changes infrequently.
>
> If you want to manually purge the cache of your site, you can do so by triggering a redeployment manually.
>
> > The cache is automatically purged upon a new deployment being created. If you ever need to invalidate the CDN cache, you can always re-deploy. Further reading: [vercel.com/docs/concepts/edge-network/caching](https://vercel.com/docs/concepts/edge-network/caching).
